Doc/library/sqlite3.rst

```diff
@@ -40,51 +40,48 @@ This document includes four main sections:
 Tutorial
 --------
 
-To use the module, start by creating a :class:`Connection` object that
-represents the database.  Here the data will be stored in the
-:file:`example.db` file::
+To use the module, start by opening a connection to an SQLite database.
+We do this by using the :func:`sqlite3.connect` function.
+The returned :class:`Connection` object represents our
+connection to the database.
+In our example,
+the database will be stored in the file :file:`example.db`::
 
    import sqlite3
    con = sqlite3.connect('example.db')
 
-The special path name ``:memory:`` can be provided to create a temporary
-database in RAM.
-
-Once a :class:`Connection` has been established, create a :class:`Cursor` object
-and call its :meth:`~Cursor.execute` method to perform SQL commands::
+Notice that the file :file:`example.db` will be created implicitly
+if it does not exist.
```

Comment:
> It might also be worth mentioning explicitly that unlike other DB systems (e.g. MySQL, postgres), sqlite3 works with files.

This is already noted in the _Introduction_ section.